config/config.go

```diff
@@ -133,9 +133,11 @@ var (
 			},
 		},
 		BlockSync: BlockSync{
-			Interval:     10 * time.Second,
-			BufferSize:   50,
-			IntervalSize: 10,
+			Interval:        10 * time.Second,
+			BufferSize:      100,
+			IntervalSize:    10,
+			MaxRepeat:       3,
```

Comment:
This config basically says for block 0 and 1 can try 3 and 2 times. Does this couple with number of allowed peers? If so, need to state clearly in the comment,

```
block #, repeated times
0                3
1                2
2                1
3                1
4                1
5                ...
```

Reply:
1 block -> block interval
2 Not couple with # of peers